test/seed.rb

```diff
@@ -24,14 +24,14 @@ def self.add_wikirate_data
     Card.create! :name=>'Death Star+Force', :type=>'analysis', :subcards=> {
       '+article'  => { :content=>"I'm your father!" }
     }
-    
-    sourcepage = Card.create! :type_id=>Card::SourceID, :subcards=>{ 
+
+    sourcepage = Card.create! :type_id=>Card::SourceID, :subcards=>{
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.